android/build.gradle

```diff
@@ -18,4 +18,5 @@ android {
 
 dependencies {
     compileOnly "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
+    implementation 'androidx.exifinterface:exifinterface:1.1.0-rc01'
```

Comment:
Note: The Android Exifinterface now comes from the androidx support. This adds a bunch of additional exif tags, and fixes some compatibility issues with some devices that could corrupt the exif tags.

Reply:
@cristianocca Does it mean people with RN <0.60 apps should go with jetifier reverse ?

Reply:
You are right, people using RN < 60 should use the reverse jetifier. I actually thought a lot about it, but reading around it seems like androidx Exifinterface is much better than the android.media one. Not only the androidx one provides more features, it also has all the security fixes that were found some time ago related to Android images.

Do you think this would be a blocker? Compat libraries support is near its end, and people should definitely be using androidx now that Android 10 is out and various phones are catching up.

Reply:
It's totally fine that it uses androidX, we will just need to add it in the README in the install section for people that uses RN < 0.60

Reply:
I haven't used the reverse jetifier myself (migrated to RN 60 a while ago and just using the regular jetifier), so that's why I didn't add any mention to it to the readme, since I'm not entirely sure how to use it and don't want to provide wrong info.

Reply:
Also just to double check, why did you put a release candidate version ? `1.10-rc01`

Reply:
That's the current latest version, the non RC one is still 1.0.0, which is outdated and missing a bunch of fixes. It would definitely good to upgrade once the final version is released. Perhaps wait  to do any merge until that one is updated? I will be having some production test of these changes next week, so I can double check of any unexpected crash coming from these changes.

Reply:
Alright seems good to me, I'll try do some QA testings on some devices in the incoming days. If we both don't face any crashes I think it can be merge to `master`. Thanks a lot @cristianoccazinsp 😀